AirCasting/MobilePeripheralSessionManager.swift

```diff
@@ -0,0 +1,73 @@
+// Created by Lunar on 07/06/2021.
+//
+
+import Foundation
+import CoreBluetooth
+
+class MobilePeripheralSessionManager {
+    private let measurementStreamStorage: MeasurementStreamStorage
+    private lazy var locationProvider = LocationProvider()
+    
+    var activeMobileSession: MobileSession?
+    var streamsIDs: [String: MeasurementStreamLocalID] = [:]
```

Comment:
`private(set)`

Reply:
How about giving a typealias for the `stream.sensorName` type? We could have a more understandable sturct with it like: `private(set) var stramsIDs: [SensorName: MeasurementStreamLocalID] = [:]`

Reply:
Okay, good idea